frontend/src/components/cookie_disclaimer.svelte

```diff
@@ -11,7 +12,7 @@
 
 {#if loadDisclaimer}
   {#if !$cookieDisclaimer}
-    <div class="flex justify-center">
+    <div class="flex justify-center" transition:fly={{ y: 200, duration: 2000 }}>
```

Comment:
I feel like 2 seconds is a bit too slow, maybe 1 second or half a second?

Reply:
I agree, changed it to 1, but changed in:transition to a fade instead. So fade in, fly out